app/controllers/MakePaymentController.scala

```diff
@@ -36,7 +36,7 @@ class MakePaymentController @Inject()(val messagesApi: MessagesApi,
   extends AuthorisedController with I18nSupport {
 
 
-  def makePayment(amountInPence: Long, taxPeriodMonth: Int, taxPeriodYear: Int): Action[AnyContent] =
+  def makePayment(amountInPence: Long, taxPeriodMonth: Int, taxPeriodYear: Int, chargeType: String, dueDate: String): Action[AnyContent] =
```

Comment:
You haven't updated the unit tests for this controller, so the tests are failing.